src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -368,6 +369,82 @@ public boolean isInGoal(int x, int y, int z) {
             logDirect("tunneling");
             return true;
         }
+        if(msg.startsWith("tunnelcustom"))
+        {
+            String suffix = msg.substring("tunnelcustom".length());
+            int a;
+            int b;
+            int c;
+            if (suffix.isEmpty()) {
+                    logDirect("Need to specify depth, height and width of the tunnel. eg .b tunnel2 depth height width");
```

Comment:
`.b` is a command added by Impact *and* the command's name is `tunnelcustom`.

Should read something like `...eg #tunnel2 depth height width`